chainer/functions/pooling/pooling_nd_kernel.py

```diff
@@ -0,0 +1,300 @@
+import chainer
+from chainer.utils.conv_nd_kernel import _map
+from chainer.utils.conv_nd_kernel import mulexp
+from chainer.utils.conv_nd_kernel import succ_sublists
+from chainer.utils.conv_nd_kernel import vars
+from chainer.utils.conv_nd_kernel import Writer
```

Comment:
Direct import of objects is not allowed in pep8 (`_map`, `mulexp`, `succ_sublists`, `vars`, `Writer`).